ElectronClient/app/gui/NoteText.jsx

```diff
@@ -1408,7 +1408,7 @@ class NoteTextComponent extends React.Component {
 			height: 30
 		};
 
-		const bottomRowHeight = rootStyle.height - titleBarStyle.height - titleBarStyle.marginBottom - titleBarStyle.marginTop - theme.toolbarHeight - tagStyle.height - tagStyle.marginBottom;
+		const bottomRowHeight = (this.state.noteTags.length !== 0) ? rootStyle.height - titleBarStyle.height - titleBarStyle.marginBottom - titleBarStyle.marginTop - theme.toolbarHeight - tagStyle.height - tagStyle.marginBottom : rootStyle.height - titleBarStyle.height - titleBarStyle.marginBottom - titleBarStyle.marginTop - theme.toolbarHeight - tagStyle.height;
```

Comment:
So instead of subtracting the `tagStyle.height` I think subtracting the `tagStyle.marginBottom` is probably the correct thing to do.

```
const bottomRowHeight = (this.state.noteTags.length !== 0) ? rootStyle.height - titleBarStyle.height - titleBarStyle.marginBottom - titleBarStyle.marginTop - theme.toolbarHeight - tagStyle.height - tagStyle.marginBottom :
			rootStyle.height - titleBarStyle.height - titleBarStyle.marginBottom - titleBarStyle.marginTop - theme.toolbarHeight - tagStyle.marginBottom;
```
The rest of the code can then be left as is including the `marginTop` added to the `editorStyle` above. Subtracting the `tagStyle.height` reduces the height of the editor too much leaving too much gap at the bottom of the editor.

On a side note, adding the `editorStyle.marginTop` can also be skipped if we don't set the `tagList` to `null`.